### PR TITLE
Agency logo

### PR DESCRIPTION
**Pull request by dgonzalez**
Serves a static resource to act as a logo

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 - Added: Added "verify" secret and changed "jwt_issuer" and "certificate_audience" parameters to support third party key server
+- Added static resource served through the api to act as a logo
```
Comment (pmcgrath) [inline CHANGELOG.md]: Lets add the new ones at the top of the list - so is reversed chronological order

Comment (dgonzalez) [reply CHANGELOG.md]: on it.

File: variables.tf [hunk 2]
```diff
@@ -411,3 +411,11 @@ variable "sms_sender" {
 variable "sms_region" {
   default = ""
 }
+variable "agency_logo_path" {
+  description = "Path for the agency logo file - will be uploaded to s3"
+  default     = ""
+}
+variable "agency_logo_s3_key" {
```
Comment (pmcgrath) [inline variables.tf:418]: Any reason we need this - why would we need to vary ?

Comment (dgonzalez) [reply variables.tf:418]: not at all.

**Comment by pmcgrath** [comment]
@dgonzalez 
Have decided to go with a static public s3 bucket

Created content at https://github.com/nearform/covid-tracker-utilities/tree/master/nearform-services-account/modules/covid-tracker-agency-logos

This is using the nearform AWS account rather than the nearform-services account which was not ready at the time
- would like to move this bucket

**Pull request closed without merging**